thirdeye/thirdeye-frontend/app/styles/pods/loading.scss

```diff
@@ -0,0 +1,5 @@
+.route-loader-container {
+  position: absolute;
+  left: 48%;
```

Comment:
couldn't we achieve the same layout with flex box ? 
Personally not a big fan of magic numbers to position items. if you were trying to offset the spinner's own height and width. You could keep both `left` and `top` at 50%, and then use the transform() trick.
`transform: translateX(-50%) translateY(-50%);`

Reply:
i added the flexbox.

Reply:
DONE.